capstone/src/main/java/com/google/sps/servlets/EventListing.java

```diff
@@ -1,18 +1,119 @@
 package com.google.sps.servlets;
 
+import com.google.api.client.auth.oauth2.Credential;
```

Comment:
Follow the instructions [here](https://github.com/google/google-java-format#using-the-formatter) to download and run the java formatter to clean up the alignment on this file.